LCD-25561 - Automate Liferay Sample Custom Element 1 scenario

joshchong opened the pull request:
https://issues.liferay.com/browse/LCD-25561

brianikim committed:
COMMERCE-11017 Remove Commerce Account usage in headless-commerce-machine-learning-impl

brianikim committed:
COMMERCE-11017 Remove Commerce Account Usage in headless-commerce-punchout

brianikim committed:
COMMERCE-11017 Remove Commerce Account usage in headless-commerce-admin-catalog-impl

brianikim committed:
COMMERCE-11017 Remove Commerce Account usage in headless-commerce-admin-order-impl

brianikim committed:
COMMERCE-11017 Remove Commerce Account usage in headless-commerce-admin-pricing-impl

brianikim committed:
COMMERCE-11017 Remove Commerce Account usage in headless-commerce-delivery-catalog-impl

brianikim committed:
COMMERCE-11017 build.gradle

brianikim committed:
COMMERCE-11017 Rename *AccountGroup in .yaml to avoid class conflicts

brianikim committed:
COMMERCE-11017 Refactor needed constants and methods to Account*

dsitu committed:
COMMERCE-11017 Minor fixes

brianikim committed:
COMMERCE-11017 Remove Commerce Account usage in headless-commerce-admin-account-impl

brianikim committed:
COMMERCE-11017 Review CommerceAccount* removal

dsitu committed:
COMMERCE-11017 Fix new test

dsitu committed:
COMMERCE-11017 Fix groupId issue

ccorreagg committed:
LPS-182579 Use removeLast instead

ccorreagg committed:
LPS-182579 Add more tests. Testing the related object entries in one to many relationships from the many side

ccorreagg committed:
LPS-182579 Simplify

sergiojimcos committed:
LPS-182213 Add all available actions in Object Entry OpenAPI

ccorreagg committed:
LPS-182579 Add test for one to many and many to many relationships using nestedFieldsDepth

sergiojimcos committed:
LPS-182213 Add baseUriInfo in OpenAPIContext

sergiojimcos committed:
LPS-182213 Set baseUriInfo in OpenAPiContext to use it for the href action values

sergiojimcos committed:
LPS-182213 Add test case exposing all available actions including an Object Action

ccorreagg committed:
LPS-182579 Add tests to validate the related System Object Entries, being returned as nested fields

sergiojimcos committed:
LPS-182213 baseline

ccorreagg committed:
LPS-182579 Refactor

sergiojimcos committed:
LPS-182213 Add new feature flag

Commit:
LPS-182621  Transform the static DLWebComponentProvider into a Util class using a Snapshot field

sergiojimcos committed:
LPS-182329 Add all available actions in Object Collection OpenAPI

Commit:
LPS-182621 Transform the static component DLViewFileVersionDisplayContextUtil into a Util class using a Snapshot field

Commit:
LPS-182376 Remove NavItemClassNameIdUtil and use ClassNameLocalServiceUtil to get ClassNameId directly

sergiojimcos committed:
LPS-182329 Add test case exposing all available collection actions in Objects OpenAPI schema

ccorreagg committed:
LPS-182579 Move method to a util class

sergiojimcos committed:
LPS-182329 Create ObjectActionTestUtil for adding new action

sergiojimcos committed:
LPS-182329 Refactor test cases

Commit:
LPS-182376 Remove WikiEngineInputEditorCommonComponentProvider, this class is not used

Commit:
LPS-182376 Transform the static component DLMimeTypeDisplayContextUtil into a Util class using a Snapshot field

Commit:
LPS-182621 Transform the static component DLItemSelectorCriterionCreationMenuRestrictionUtil into an Util class initializing the Service tracker into a static block instead of an activate method

Commit:
LPS-182621 Add dependencies

Commit:
LRAC-13511 Update macro to accept type and filter to be used in data report API

Commit:
LRAC-13511 Rename the macro and improve the failure message

Commit:
LRAC-13511 Move the CreateQueryDataWithAnalyticsCloudAPI test to the Data Report file

Commit:
LRAC-13511 Add RequestShowErrorMessageWhenUseInvalidType testcase

Commit:
LRAC-13511 SF

brianikim committed:
COMMERCE-11017 gw buildREST (Autogenerated)

ccorreagg committed:
LPS-182579 Set the original NestedFieldsContext

smottal committed:
COMMERCE-11344 Fix override hidden panel

Commit:
LPS-182621 Turning methods into static so you don't have the need to create an instance of them

ccorreagg committed:
LPS-182579 Add assertion to identify the error

julschong committed:
LPS-178022 Remove Optional usages in CustomFacetDisplayContextBuilder

julschong committed:
LPS-178022 Reduce Optional usage to constructor only in CustomFilterDisplayContextBuilder

julschong committed:
LPS-178022 Remove unused methods in SearchStringUtil and SearchOptionalUtil

julschong committed:
LPS-178022 Remove Stream usages in SearchStringUtil

julschong committed:
LPS-178022 Use GetterUtil.getString() instead of Optional.orElse()

david-truong committed:
LPS-182960 add test to check performance of discarding changes in a publication

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

julschong committed:
LPS-178022 Apply to usages

julschong committed:
LPS-178022 These methods return int value, Optional.orElse() will never happen

allen-ziegenfus committed:
LCD-26596 Update refresh script to always use latest 7.4 version when refreshing

brianikim committed:
COMMERCE-11386 We should first check whether a CIBookedQuantity exists before restocking

itstali committed:
LPS-163191 Add adjust for typeAllNewApplicationDriverInfoFields macro

itstali committed:
LPS-163191 Add CanAccidentCitationBeAdded test

allen-ziegenfus committed:
LCD-26596 Run refresh_liferay_sample_workspace.sh script

tainguyen100 committed:
LCD-26694 Add Liferay Sample Custom Element 2 smoke test

joshchong committed:
LCD-25561 Automate Liferay Sample Custom Element 1 scenario

brianikim committed:
COMMERCE-11017 SF
Needed because of: https://issues.liferay.com/browse/LPS-180838

gamerson committed:
LPS-181325 update spring-boot sample ids to distiguish from node sample

gamerson committed:
LPS-181325 update object actions for spring-boot and add node actions

gamerson committed:
LPS-181325 remove unneeded

pietrobocale committed:
COMMERCE-11344 Add new paths

pietrobocale committed:
COMMERCE-11344 Add new macros

pietrobocale committed:
COMMERCE-11344 Add new test CanUseVirtualSkuOverride

Mikellorza committed:
LPS-183651 Build service layout-seo-service

Mikellorza committed:
LPS-183651 SF to follow source/target pattern in copy methods of layout-utility-page-service

Mikellorza committed:
LPS-183651 Build service layout-utility-page-service

Mikellorza committed:
LPS-183651 SF to follow source/target pattern in copy methods of style-book-service

Mikellorza committed:
LPS-183651 Build service style-book-service

matuzalemsteles committed:
LPS-183138 Update packages `@clayui/*` to v3.94.0

itstali committed:
LPS-163191 Add CanViewApplicationStatusAsUnderwriting test

Commit:
LPS-182621 Moving DLFileEntryTypeIconProviderUtil logic to MenuItemProvider

matuzalemsteles committed:
LPS-183138 regen snapshot

Commit:
LPS-182621 Deleting DLFileEntryTypeIconProviderUtil

Commit:
LPS-182621 removing references the class DLFileEntryTypeIconProviderUtil

Commit:
LRAC-12949 Add RequestCommandHasInvalidType testcase

Commit:
LRAC-13592 Add InternalProblemOccurredInExportProcess testcase

Commit:
LRAC-13382 Update the type used in the API not use accounts which is an invalid type

Commit:
LRAC-12949 SF

tinatian committed:
LPS-178022 SF, simplify

tinatian committed:
LPS-178022 SF, inline

Commit:
LPS-182656 Simplify by directly using map, remove the transformation to KeyValuePair to avoid creating unnecessary instance, and the keys will not have duplicates, as they will be combined into a single URL

Commit:
LPS-182656 Replace Stream with for loop in class DDMRESTDataProvider

Mikellorza committed:
LPS-183651 SF to follow source/target pattern in copy methods of portal-impl

Mikellorza committed:
LPS-183651 Build service portal-impl

Commit:
LPS-160277 Add file path to home page and add paths for the footer

Commit:
LPS-160277 Add file macro to home page and add macros for the footer

Commit:
LPS-160277 Add test AssertThatTheCookiePolicyPageOpens

Commit:
LPS-160277 Add test AssertThatTheLanguageLinksWork

ealonso committed:
LPS-183823 Those exceptions are never thrown

ealonso committed:
LPS-183823 Uses dynamic data mapping module classes

ealonso committed:
LPS-183823 Uses reference

ealonso committed:
LPS-183823 Copy logic from StorageEngineManager, to avoid dependencies with old DDM in portal kernel

ealonso committed:
LPS-183823 Apply changes

ealonso committed:
LPS-183823 No needed

ealonso committed:
LPS-183823 Uses new class

ealonso committed:
LPS-183823 Unnecessary dependency

ealonso committed:
LPS-183823 baseline

JanisZhang committed:
LPS-183716 Make ServletContextUtil non-component

brianikim committed:
COMMERCE-11017 gw baseline (Autogenerated)

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Sort

brianchandotcom committed:
LPS-182213 Sort

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Inline

brianchandotcom committed:
LPS-182213 Only used once

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Sort

brianchandotcom committed:
LPS-182213 Sort

brianchandotcom committed:
LPS-182213 Sort

brianchandotcom committed:
LPS-182213 Inline

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 Rename

ccorreagg committed:
LPS-182579 Move random generation to the utils, create update method and use the utils from different tests

ccorreagg committed:
LPS-182579 Rename

Commit:
LPS-183831 Add missing step

sergiojimcos committed:
LPS-182213 Rename

brianikim committed:
COMMERCE-11017 SF. Rename method

brianikim committed:
COMMERCE-11017 gw buildService (Autogenerated)

tainguyen100 committed:
LCD-26695 Add Liferay Sample Global CSS smoke test

tainguyen100 committed:
LCD-26697 Add Liferay Sample IFrame 1 - Counter App smoke test

manuelecastro committed:
LPS-166193 Remove SessionID from audit log

thaynam committed:
LPS-183644 Add status values to mdf request form

thaynam committed:
LPS-183644 Add status values to mdf claim form

thaynam committed:
LPS-183644 Add status values to deal reg form

victorhcunha committed:
LPS-169755 Fix SCHEDULER_VIEW_MONTH_DATE Path

manuferreira committed:
LRAC-13479 Update createDataReport macro excluding the '$' sign of the JSONCurl because it is returning a response with some of the properties not in the string format

manuferreira committed:
LRAC-13497 Add assertApiResponse macro

manuferreira committed:
LRAC-13497 Add searchApiResponse macro

manuferreira committed:
LRAC-13497 Add createIndividualOrAssetDataReport macro

abhnerramos committed:
LPS-183789 Adding stubs for DatasetPagination testcase

manuferreira committed:
LRAC-13497 Add RequestToAccessDataForAllBlogs testcase

manuferreira committed:
LRAC-13497 Add RequestToAccessDataForAllIndividuals testcase

manuferreira committed:
LRAC-13497 Add RequestToAccessDataForSpecificBlog testcase

manuferreira committed:
LRAC-13497 Add RequestToAccessDataForSpecificIndividual testcase

manuferreira committed:
LRAC-13497 SF

Commit:
LPS-183895 Fix MyWorkflowTasks tests

victorhcunha committed:
LPS-169755 Remove ViewCorrectNumberOfWeeksMonthView from quarantine

ealonso committed:
LPS-183972 Removes feature flag

melodywu committed:
LRQA-80397 ignore the testcases and remove four of them

Commit:
LPS-182656 Simplify, merge methods _getAdditionalParameters, _getFilterAndPaginationParameters and _getQueryParameters into one method _getAllParameters to reduce the number of operations on map

Commit:
LPS-182656 Simplify, directly use treeMap to reduce the number of operations on map, because every time method _getData is executed, variable allParametersMap must be sorted in method _getPortalCacheKey, and method jsonWebServiceClient.doGet does not care about the order of allParametersMap

manuferreira committed:
LRAC-13497 Rename GenerateAPI file to ExportAPI

liulukiwi committed:
LPS-183848 Modify path to make customization button secondary text to be found

Commit:
LPS-183630 Add Commerce Channel

Commit:
LPS-183630 Add Commerce Catalog Options

Commit:
LPS-183630 Add MP Accounts and User Accounts

Commit:
LPS-183630 Add User Role Permissions

joseabelenda committed:
LPS-183630 Rename

ealonso committed:
LPS-183029 Adds new test to check the correct behavior of _getCollectionFieldsJSONObject when we have different segments experiences.

Having a collection with different variations in edit mode for the selected experience we should show:

- If there is an asset list variations matching with the selected segment of the segments experience, we should show the asset list variation.
- If there is not an asset list variations matching with the selected segment of the segments experience, we should show the default asset list variation.

raquelpaiva committed:
LPS-183458 Add Breadcrumb on Home page

raquelpaiva committed:
LPS-183458 Add Breadcrumb on Payment page

ealonso committed:
LPS-183029 We should assign the var to the result of add

raquelpaiva committed:
LPS-183458 Add Breadcrumb on Request and Request form pages

abhnerramos committed:
LPS-183521 Update deleteAllDatasetEntries position on DatasetsAPI's tearDown

abhnerramos committed:
LPS-183521 Update deleteAllDatasetEntries position on FDSViews' tearDown

abhnerramos committed:
LPS-183521 Update deleteAllDatasetEntries position on FDSEntry's tearDown

ealonso committed:
LPS-183029 We don't need to calculate it two times

ealonso committed:
LPS-183029 Creates DefaultLayoutListRetrieverContext only when is necessary

ealonso committed:
LPS-183029 Reorder

ealonso committed:
LPS-183029 Simplify

ealonso committed:
LPS-183029 We should set segmentEntryIds before call count

ealonso committed:
LPS-183029 Check first for the selected segments experience and then for the default one, if we don't have results

kevhlee committed:
LPS-175368 Only enable on master for now, since FeatureFlagManagerUtil
is not available in 7.x branches

Commit:
LPS-182596 portal-search-test: Fix testTitleIndexedInTriplicateDistortsScores

https://issues.liferay.com/browse/LPS-182596

thaynam committed:
LPS-183644 Change status proo name to submitted

kevhlee committed:
LPS-175368 Enable for SF tests as well

thaynam committed:
LPS-184082 Rename function names in MDF Request and Claim cancel button

danvb112 committed:
LPS-184110 Refactor getAppListProductSpecifications function and fix API URL

brookedalton committed:
LPS-184108 Use JSON macros where applicable in ClassicSearch testcase

michaelhashimoto committed:
LRCI-3588 Only run deployPoshi if the client-extension.poshi.yaml exists

joshchong committed:
LCD-26696 Add Liferay Sample Global JS smoke test

joshchong committed:
LCD-26698 Add Liferay Sample Theme CSS 1 smoke test

joshchong committed:
LCD-26699 Add Liferay Sample Theme Favicon smoke test

joshchong committed:
LCD-26700 Add Liferay Sample Theme Spritemap 1 smoke test

joshchong committed:
LCD-27073 Add Virtual Instance counterpart tests

Commit:
LRQA-80455 Fix typo

Commit:
LRQA-80534 Increase timeout explicit wait to make case stable

shuyangzhou committed:
LPS-184192 Remove verifyDB function, since LPS-68837 2c6b943a2370b9ff97d81bc4bf503f11356e4197 this function has been practically removed. Because upgrade framework always ensures InitialUpgradeStep running with Release creation to target version. It is not possible to see a missing release entry or release with schema 0.0.0 for a service builder module at runtime, therefore this verifyDB() always does nothing.

shuyangzhou committed:
LPS-184192 Build service

shuyangzhou committed:
LPS-184192 Remove from UI

shuyangzhou committed:
LPS-184192 Remove language key

shuyangzhou committed:
LPS-184192 buildLang

brianchandotcom committed:
COMMERCE-11386 Buy space

shuyangzhou committed:
LPS-184192 Remove test

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/dxp/apps/osb/osb-faro/.gitrepo'.

Commit:
LPS-183991 Remove setup virtual instance and tear down block from Pages Tree

brianchandotcom committed:
LPS-181325 Remove it here too then

brianchandotcom committed:
LPS-181325 Rename

brianchandotcom committed:
LPS-181325 Consistency

brianchandotcom committed:
LPS-181325 This extension is not even using Spring Boot

brianchandotcom committed:
LPS-181325 Rename for consistency

brianchandotcom committed:
LPS-181325 auto SF

dantewang committed:
LPS-182656 SF, rename

brianchandotcom committed:
LPS-182973 Rename

brianchandotcom committed:
LPS-182973 Rename

brianchandotcom committed:
LPS-182973 Sort

brianchandotcom committed:
LPS-182973 Rename

brianchandotcom committed:
LPS-182973 I don't see it in the other timers

brianchandotcom committed:
LPS-182973 Rename

Jeremy-Ch committed:
LPS-184117 Navigate to the blog entry by UI

brianchandotcom committed:
LPS-182973 Sort

ccorreagg committed:
LPS-182579 sort

brianchandotcom committed:
LCD-26596 Sort

brianchandotcom committed:
LPS-182579 Wordsmith

brianchandotcom committed:
LPS-182579 Should this be plural or singular? I'm just matching the comments.

brianchandotcom committed:
LPS-182579 Rename

brianchandotcom committed:
LPS-182579 Consistency with other comments in this class

brianchandotcom committed:
LPS-182579 Rename

brianchandotcom committed:
LPS-182579 auto SF

Commit:
LPS-183991 Modify macro to make it reusable

michaelhashimoto committed:
LRCI-3588 Add ability to start & stop spring boot applications for workspaces

michaelhashimoto committed:
LRCI-3588 Add custom scripts to start & stop spring boot applications for liferay-sample-workspace

Jeremy-Ch committed:
LPS-182825 Add property to enable the specific virtual host

Commit:
LPS-183991 Sort

liferay-translation committed:
LPS-77699 Update Translations

naraandrade committed:
LPS-184110 Remove misplaced text on svg

raquelpaiva committed:
LPS-183458 Add breadcrumb on Report and Report request

brianchandotcom committed:
LPS-182213 Rename

brianchandotcom committed:
LPS-182213 We're not even returning anything?

brianchandotcom committed:
LPS-182213 Match: Map<String, Schema> actionSchemas

abhnerramos committed:
LPS-184162 Add ignore on tests

silasbsi committed:
LPS-181959 Commerce products does not get returned after calling an endpoint

thaynam committed:
LPS-184171 Change MDF status change to patch

shuyangzhou committed:
LPS-184192 Breaking changes

brookedalton committed:
LPS-184108 Close iframe

manuferreira committed:
LRAC-13497 Update RequestShowErrorMessageWhenUseInvalidType testcase

manuferreira committed:
LRAC-13497 Update assertApiResponse macro

manuferreira committed:
LRAC-13497 Update searchApiResponse macro

manuferreira committed:
LRAC-13497 Update viewApiResponseForRootEndpoint macro

manuferreira committed:
LRAC-13497 Update RequestToAccessDataForAllBlogs testcase

manuferreira committed:
LRAC-13497 Update RequestToAccessDataForAllIndividuals testcase

manuferreira committed:
LRAC-13497 Update RequestToAccessDataForSpecificBlog testcase

manuferreira committed:
LRAC-13497 Update RequestToAccessDataForSpecificIndividual testcase

manuferreira committed:
LRAC-13497 Add code to start the container in the teardown

manuferreira committed:
LRAC-13497 SF

joshchong committed:
LCD-27021 Add test for asserting startup logs and CX admin

joshchong committed:
LCD-27074 Run Liferay Sample Workspace tests in relevant when changes made to workspaces/liferay-sample-workspace
